### ISSUE TITLE
Add Close / Open Window Flame Animation

### ISSUE THREAD

**Issue by GlitchyCrafting** [WillPower3309/swayfx#45]
Could the fire animation in Wayfire, or at least something similar, be ported to SwayFX?

**Comment by WillPower3309** [comment]
Yup, absolutely! Not a priority right now as we'd need some base animation groundwork done (which could be easily done as part of the fade in / out animation issue), but open to PRs.
